### PR TITLE
Fix randomly failing hybrid test

### PR DESCRIPTION
**Pull request by donomii**
### Review checklist

- [ ] Documentation has been updated, if necessary. Link to changed documentation:
- [ ] Chaos pipeline run or not necessary. Link to pipeline:
- [ ] All new code is covered by tests where it is reasonable.
- [ ] Performance tests have been run or not necessary.

### CODE REVIEW

File: usecases/traverser/hybrid/hybrid_fusion.go [hunk 1]
```diff
@@ -28,7 +28,7 @@ func FusionRanked(weights []float64, resultSets [][]*search.Result, setNames []s
 			}
 			tempResult := res
 			docId := tempResult.ID
-			score := weights[resultSetIndex] / float64(i+60) // TODO replace 60 with a class configured variable in the schema
+			score := float32(weights[resultSetIndex] / float64(i+60)) // TODO replace 60 with a class configured variable in the schema
```
Comment (moogacs) [inline usecases/traverser/hybrid/hybrid_fusion.go:31]: **question:** shall it be `float64` ?

Comment (moogacs) [inline usecases/traverser/hybrid/hybrid_fusion.go:31]: **question (non-blocking):** is it intended to be `float32` not  `float64` ?

File: adapters/repos/db/ranked_fusion_test.go [hunk 2]
```diff
@@ -364,52 +404,6 @@ func TestRFJourney(t *testing.T) {
 		require.Equal(t, float32(0.016393442), results[1].Score)
 	})
 
-	// Check basic search with one property
-	results_set_1, err := repo.VectorSearch(context.TODO(), dto.GetParams{
```
Comment (moogacs) [inline adapters/repos/db/ranked_fusion_test.go:368]: **question (non-blocking):** is it intended to be deleted ?

File: usecases/traverser/hybrid/hybrid_fusion.go (hunk 1 above)
Comment (donomii) [reply usecases/traverser/hybrid/hybrid_fusion.go:31]: yes, everything else is stored in float32, so this has to be float32 as well.  I should probably switch the weights as well, but this works for now

File: adapters/repos/db/ranked_fusion_test.go (hunk 2 above)
Comment (donomii) [reply adapters/repos/db/ranked_fusion_test.go:368]: yes, this is the failing test.  I replaced it with the smaller test above

**Review by dirkkul — APPROVED** [review]

**Pull request merged**